lib/spack/docs/packaging_guide.rst

```diff
@@ -2016,45 +2016,133 @@ the Python extensions provided by them: once for ``+python`` and once
 for ``~python``.  Other than using a little extra disk space, that
 solution has no serious problems.
 
------------------------------------
-Implementing the ``install`` method
------------------------------------
+.. _installation_procedure:
 
-The last element of a package is its ``install()`` method.  This is
+---------------------------------------
+Implementing the installation procedure
+---------------------------------------
+
+The last element of a package is its **installation procedure**.  This is
```

Comment:
I feel like this section, though probably the most important one, has gotten buried in the packaging guide.  I wonder if there's a way to bring it closer to the top and make it more obvious.  Or maybe the packaging guide needs to be a top-level section...